app/components/Form/ApplicationForm.tsx

```diff
@@ -292,6 +298,9 @@ const ApplicationForm: React.FC<Props> = ({
   const handleDisabled = (page: string, noErrors: boolean) => {
     let disabled = false;
     switch (true) {
+      case isWithdrawn:
+        disabled = false;
```

Comment:
Sonarcloud flagged this as redundant, we can remove the disabled assignment here and just break.

Reply:
Alright I refactored this and removed the let altogether, sonar looks happy now.